muesli/web/templates/user/login.pt

```diff
@@ -11,6 +11,10 @@ Sie sind zur Zeit als ${request.user.email} angemeldet.
   pyramid.security glaubt, sie seien ${user}.
 </p>
 
+<p>
+MÜSLI verwaltet viele Übungsgruppen und Vorlesungen an der Fakultät für Mathematik und Informatik der Universität Heidelberg. Diese Version von MÜSLI wurde in den Semesterferien zum Wintersemester 2011/12 neu aufgesetzt.
```

Comment:
Please add line breaks here, so each line has only 80 characters. This should not change the output in the browser, but helps readability. Unfortunately this was ignored in large parts of the code base :)

Reply:
Well, that was an easy change. Fixed it :D